File: abc083/abc083b.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        int a = sc.nextInt();
        int b = sc.nextInt();

        int result = 0;
        for(int i = 1; i <= n; i++) {
            int sum = 0;
            int j = i;
            while (j != 0) {
              sum += j % 10;
              j /= 10;
            }
            if ((a <= sum) && (sum <= b)) {
                result += i;
            }
        }
        System.out.println(result);
    }
}

File: abc083/abc083b.go
package main

import (
	"fmt"
)

func main() {
	var n, a, b int
	fmt.Scan(&n, &a, &b)
	result := 0
	for i := 1; i <= n; i++ {
		k := 0
		for j := i; j != 0; j /= 10 {
			k += j % 10
		}
		if a <= k && k <= b {
			result += i
		}
	}
	fmt.Println(result)
}
